deployments/goerli/AssetAttributesRegistry.json

```diff
@@ -1,701 +0,0 @@
-{
-  "address": "0x29B6261A4244d2a7AAB0B3EDaa1159700C9A93Ae",
```

Comment:
We don't need this one on goerli?

Reply:
It relates to the old cats & gems on L1 so no, shouldn't need it anymore. But I have reinstated the file in case you need to look anything up there.

Reply:
I wasn't sure if this was a new deployment since I couldn't find it for mainnet

Reply:
No, the new one is PolygonAssetAttributesRegistry